gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -469,7 +469,9 @@ def add_arguments(self, parser):
     parser.add_argument(
         '--output_file',
         required=True,
-        help='The full path of the VCF file to store the result.')
+        help='The full path of the VCF file to store the result. If run with '
+             'Dataflow, a path in Google Cloud Storage that your project has '
```

Comment:
nit: ... Cloud Storage is required.

I think it's implied that they have to have write access to the bucket.